client/src/includes/initTooltips.ts

```diff
@@ -56,8 +59,8 @@ const hideTooltipOnBreadcrumbExpandAndCollapse = {
   },
 };
 
-/*
- If the toggle button has a toggle arrow, rotate it when open and closed
+/**
+ * If the toggle button has a toggle arrow, rotate it when open and closed
  */
 const rotateToggleIcon = {
   name: 'rotateToggleIcon',
```

Comment:
The line below, let's add a type reference as you have done for other similar lines.

```
  fn(instance: Instance) {
```